NitroxClient/MonoBehaviours/Gui/InGame/LostConnectionModal.cs

```diff
@@ -80,5 +102,31 @@ private IEnumerator Show_Impl()
             yield return null;
             IngameMenu.main.ChangeSubscreen(SUB_WINDOW_NAME);
         }
+
+        private static uGUI_TextFade MakeLostConnectionText(string text)
+        {
+            GameObject gameObject = Instantiate(loadingTextGameObject, loadingTextGameObject.transform.parent);
+            gameObject.name = "LoadingScreenLostConnection";
+
+            uGUI_TextFade textFade = gameObject.GetComponent<uGUI_TextFade>();
+            textFade.SetColor(Color.red);
+            // We jump 2 lines so that it's written after the versison text
+            textFade.SetText($"\n\n{text}");
+            textFade.SetAlignment(TextAnchor.UpperRight);
+            textFade.FadeIn(1f, null);
+
+            return textFade;
+        }
```

Comment:
Afaik the LostConnectionModel is unavoidable (you have to click it). Therefor this text is imo not providing more info than there already is and can be deleted.